### PR TITLE
Changes to the query page

### PR DESCRIPTION
**Pull request by mik-0**
# Type of change

Update

#### Does it contain breaking changes?

No

# Description

Due to a change in the searchseco-api, a GitHub personal access token is no longer required to query. The field has been removed.
We can now specify a branch for our query through a new field.

𝕨𝕖 𝕤𝕖𝕔𝕦𝕣𝕖 𝕤𝕖𝕔𝕠𝕤.

### CODE REVIEW

**Comment by mik-0** [comment]
Please do **not** merge this pull request yet! The searchseco-api dev branch has not been merged with main currently.

File: src/pages/Query.tsx [hunk 1]
```diff
@@ -205,26 +196,22 @@ const Query = () => {
...
+                  {...register('branch', {
+                    required: false,
                   })}
                   type="text"
-                  placeholder="Your Github token"
-                  id="token"
-                  aria-invalid={errors.token ? 'true' : 'false'}
-                  error={errors.token}
-                  defaultValue={storedToken}
+                  placeholder="master"
```
Comment (ThijnK) [inline src/pages/Query.tsx:210]: Do you _want_ to be cancelled??

File: src/pages/Query.tsx [hunk 2]
```diff
@@ -205,26 +196,22 @@ const Query = () => {
...
+                  id="branch"
+                  aria-invalid={errors.branch ? 'true' : 'false'}
+                  error={errors.branch}
+                  defaultValue="master"
```
Comment (ThijnK) [inline src/pages/Query.tsx:214]: As we discussed, we probably do not want defaultValue, should be `undefined` by default

File: src/pages/Query.tsx [hunk 3]
```diff
@@ -423,9 +410,7 @@ const ExplanationButton = () => {
               <ol className="list-decimal pl-5">
                 <li>
                   Enter the URL of the GitHub repository you want to query. You
-                  also need to enter your GitHub token to be able to query the
-                  database. This token will be used to download the repository.
-                  Only read access is required.
+                  can also specify the branch.
```
Comment (ThijnK) [inline src/pages/Query.tsx:413]: Let's make it:

"Enter the URL of the GitHub repository you want to query. Optionally, you may specify the branch of the repository to query. By default, the main branch is queried."

**Review by ThijnK — APPROVED** [review]

**Pull request merged**